### PR TITLE
Bump Prettier version

### PR DESCRIPTION
**Pull request by dprokop**
FYI @davkal

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
     "postcss-browser-reporter": "^0.5.0",
     "postcss-loader": "^2.0.6",
     "postcss-reporter": "^5.0.0",
-    "prettier": "1.9.2",
+    "prettier": "^1.9.2",
```
Comment (davkal) [inline package.json]: Especially with formatters, I'd fix the version. Should we bump it here manually to `1.16.4` ?

Comment (dprokop) [reply package.json]: You are right, we can fix it here. Will limit the amount of confusion related to prettier changes. Updated in db6f84d61423f6cb9e0d02782a17a225450be2b8

**Comment by davkal** [comment]
Looks good. Only concern: Is now a good time for changing 80+ files in master @torkelo ?

**Comment by torkelo** [comment]
no, it's been hard cherry picking as it is, this will make it even hard for 6.0 fixes. 

So would have to cherry pick this as well, which will probably work fine but not sure.

**Comment by torkelo** [comment]
ok, tested and this could be cherry-picked without any big issues (one conflict) issue to v6.0.x

**Comment by torkelo** [comment]
This was squashed cherry pick this commit, https://github.com/grafana/grafana/commit/88a46e6dd4edd6af454e42e5b8c6ed5876349d9a 

There will be 1 conflict (1 deleted file, and one file that does not exist in v6.0.x) , I can do it or help

**Pull request merged**